crates/ty_python_semantic/src/types.rs

```diff
@@ -9200,7 +9200,7 @@ impl<'db> BoundSuperType<'db> {
 // Make sure that the `Type` enum does not grow unexpectedly.
 #[cfg(not(debug_assertions))]
 #[cfg(target_pointer_width = "64")]
-static_assertions::assert_eq_size!(Type, [u8; 24]);
+static_assertions::assert_eq_size!(Type, [u8; 16]);
```

Comment:
Nice @ibraheemdev some of your recent changes reduced the size of `Type` again!

Reply:
Huh, that's interesting. The only change to the stack types was that `Id` is now aligned to 4-bytes instead of 8.